### DISCUSSION TITLE
Custom Camera Source?

### DISCUSSION THREAD

**Discussion by mysticprysm** [OrcaSlicer/OrcaSlicer#1586] [Ideas]
Best way I can word this, is maybe add a way to change the camera feed to a URL you can input, like an IP camera. Similar to on octoprint, the video URL box can technically display ANY video url. I've made a cold shoe mount for the P1P, and can attach any phone or camera with it, and if there could be a way to monitor an IP stream instead of the built in camera, it could make for VERY versatile setups.

**Comment by mysticprysm** [comment]
Anyone know if there's a way this could be easily implemented? Even if I had to compile my own version?

**Reply by jbowers01** [reply]
I would love to see this implemented
